Formula/dcled.rb

```diff
@@ -23,7 +23,6 @@ class Dcled < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "c0c7a6b33e48eb8e369235f0bd1e84d10c161a41eaec0ab93854bec94df8aee0"
   end
 
-  depends_on "libhid"
```

Comment:
Not sure if we need revision bump/bottles. I don't think existing bottles use `libhid`.

From `README`:
> As of dcled version 2.1, you no longer need libhid, but you are going to need a
copy of the libusb-1.0 dev files in order to build dcled.  If you are running
debian or ubuntu, try running 'apt-get install libusb-1.0-0-dev' to get the
right files installed.

---

Yosemite bottle:
```console
❯ otool -L bin/dcled
bin/dcled:
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1213.0.0)
	@@HOMEBREW_PREFIX@@/opt/libusb/lib/libusb-1.0.0.dylib (compatibility version 2.0.0, current version 2.0.0)
```

Linux bottle:
```console
❯ objdump -p bin/dcled | grep NEEDED
  NEEDED       libm.so.6
  NEEDED       libusb-1.0.so.0
  NEEDED       libc.so.6
```